asciidoc/courses/llm-fundamentals/modules/1-introduction/lessons/1-neo4j-and-genai/lesson.adoc

```diff
@@ -0,0 +1,76 @@
+= Introduction to Neo4j & GenAI
+:order: 1
+
+In this course, we will cover everything you need to know to use Neo4j effectively with Generative AI.
+You will learn to all about Large Language Models (LLMs), Knowledge Graphs and how they can be used together effectively.
+
+So we start on the same page, let's first cover the basics.
+
+== What is Neo4j?
+
+Neo4j is a graph database management system that allows us to store, manage, and query highly connected data.
+Unlike traditional relational databases, which use tables and rows, Neo4j uses a graph-based model with nodes and relationships.
+This makes it particularly well-suited for representing and querying complex, interconnected data.
+
+
+== What are Knowledge Graphs?
+
+Knowledge Graphs are a specific implementation of a Graph Database, where the underlying data is designed to capture and represent knowledge with a rich semantic context.
+They not only capture the entities and connections between, but also the meaning behind them.
+
+Knowledge graphs are tailored for semantic search, data retrieval, reasoning, and data amalgamation, frequently powering search engines, AI, and research.
+On the other hand, Neo4j's regular graphs serve diverse roles, such as social networking, recommendations, and fraud detection, not always demanding semantic depth.
+
+Knowledge graphs typically lean on ontologies, offering structured concepts within a domain and their interrelations. Conversely, while Neo4j can engage with ontologies and offers schema constraints, its standard graphs may not consistently utilize or demand structured semantic frameworks.
+
+With their semantic foundation, knowledge graphs can readily adapt and evolve as they assimilate new information and face domain shifts. In contrast, Neo4j, with its schema-free design, emphasizes flexibility in data accommodation rather than adjustments based on changing knowledge or semantics.
```

Comment:
Don't do a KG vs. Neo4j
Neo4j is a database for KGs, period
the ontology faction just wants to make it sound as if you need an Ontology for a KG, let's not follow their lead.